src/+io/bin_to_csv.m

```diff
@@ -91,6 +91,10 @@ function bin_to_csv(fnames, n_col, ns, options, SoilLayer, GroundwaterSettings)
     Sim_qtot_units = repelem({'cm s-1'}, length(depth));
     write_output(Sim_qtot_names, Sim_qtot_units, fnames.Sim_qtot_file, n_col.Sim_qtot, ns, true);
 
+    % Comment unnecessary large size files
+    %{
+    write_output({'Bottom of canopy irradiance in the shaded fraction, and average BOC irradiance'}, {'First 2162 columns: shaded fraction. Last 2162 columns: average BOC irradiance. Unit: Wm-2 um-1'}, ...
+            fnames.BOC_irradiance_file, n_col.BOC_irradiance, ns, true);
```

Comment:
Please remove these lines instead of commenting out, the same in other places in this script. Also, avoid using documentation syntax i.e. `%{ %}` for commenting out inside the code.

Reply:
> Please remove these lines instead of commenting out, the same in other places in this script. Also, avoid using documentation syntax i.e. `%{ %}` for commenting out inside the code.

Okay, I removed all the commented lines